examples/flutter_firestore_todos/lib/blocs/todos/todos_bloc.dart

```diff
@@ -18,8 +18,8 @@ class TodosBloc extends Bloc<TodosEvent, TodosState> {
 
   final TodosRepository _todosRepository;
 
-  Future<void> _onLoadTodos(LoadTodos event, Emitter<TodosState> emit) {
-    return emit.onEach<List<Todo>>(
+  void _onLoadTodos(LoadTodos event, Emitter<TodosState> emit) {
```

Comment:
This is working as expected. `onEach` returns a Future and it's important to return the resulting future in order to properly handle things like cancelations when new events are added.